auth-web/src/components/auth/staff/ALLProductsLauncher.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-card
+    class="product-container"
+    :href="businessURL"
+  >
+    <v-row
+      align="center"
+      no-gutters
+    >
+      <v-col cols="auto">
+        <img
+          class="product-img"
+          :src="getImgUrl(img)"
+          alt="Product Image"
+        >
+      </v-col>
+      <v-col
+        class="product-info"
+        align-self="baseline"
+      >
+        <h2>{{ title }}</h2>
+        <p class="mt-5 mb-0">
+          {{ text }}
+        </p>
+        <v-btn class="primary product-info__btn px-5">
+          Open
+          <v-icon>mdi-chevron-right</v-icon>
+        </v-btn>
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs } from '@vue/composition-api'
+import ConfigHelper from '@/util/config-helper'
+import { useI18n } from 'vue-i18n-composable'
+
+export default defineComponent({
+  setup () {
+    const { t } = useI18n()
+
+    const state = reactive({
+      img: 'AssetsRegistries_dashboard.jpg',
+      title: t('viewAllProductsLauncherTitle').toString(),
+      text: t('viewAllProductsLauncherText').toString()
+    })
+
+    const businessURL = computed(() => ConfigHelper.getBusinessURL())
```

Comment:
need a business identifier

Reply:
for?

Reply:
I don't think this is right, url should be https://test.bcregistry.gov.bc.ca/dashboard/?

Reply:
`getBcrosDashboardURL`

Reply:
this is: https://test.bcregistry.gov.bc.ca/dashboard/
<img width="1210" alt="image" src="https://github.com/user-attachments/assets/50ada7f5-ef00-4e16-9a0d-81da69b5b3a7" />
but on design:
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/5c5e0846-7789-452e-ad4e-3eac790cd719" />

Reply:
thanks! updated